web/themes/custom/usagov/templates/includes/nav-benefits-block.html.twig

```diff
@@ -0,0 +1,16 @@
+{% set lang = node.langcode.value %}
+
+{% if (node.field_benefits_callout_ref) %}
+  {% set benefits_url = drupal_url('node/' ~ node.field_benefits_callout_ref.value[0].target_id ) %}
+
+  <div class="benefits-block-bg nav-benefits">
+    <div class="benefits-block">
+      <img src="/themes/custom/usagov/images/ICONS_beta_Benefits.svg" alt="Visit the benefit finder page" height="56px">
```

Comment:
The alt text here would need to be translated to Spanish. Alternatively, perhaps we don't need alt text for this image, since I think it's just decorative. 

If we don't immediately get an answer on this, we can merge it in and follow up with a correction later.

Reply:
removed alt text for decorative image